packages/splitter/src/useSplitter.ts

```diff
@@ -54,45 +70,49 @@ export interface IUseSplitterProps extends Omit<HTMLProps<any>, 'onChange'> {
   /** A callback that receives updated valueNow when a separator moves */
   onChange?: (valueNow: number) => void;
   /** A boolean that changes splitter from left-to-right to right-to-left mode */
-  rtl?: boolean;
+  placement: SplitterPlacement;
```

Comment:
Is this description still valid?

Also, throughout Garden, the concept of `placement` is RTL-aware. In other words, "end" will orient to the right in LTR environments, and to the left in RTL. Conversely, "start" is left in LTR and right in RTL. I might not be grasping the logic here, but how is this functioning in line with Garden APIs?

Reply:
I missed the refactor on this one. As discussed offline, we will be moving back to an `rtl` prop and moving `placement` to `position`. I will update these comments accordingly.

Reply:
I refactored `placement` to `position` with an enum type of `SeparatorPosition.LEADS | SeparatorPosition.TRAILS` and reintroduced an `rtl` prop to keep `position` relative to reading direction. https://github.com/zendeskgarden/react-containers/pull/405/commits/1fcd3fe466056a77363624600aee432203f87351